cdap-watchdog/src/main/java/co/cask/cdap/logging/LoggingConfiguration.java

```diff
@@ -51,7 +51,7 @@
   // Table used to store log metadata
   public static final String LOG_META_DATA_TABLE = "log.meta";
   // Defaults
-  public static final String DEFAULT_LOG_PATTERN = "%d{ISO8601} - %-5p [%t:%C{1}@%L] - %m%n";
+  public static final String DEFAULT_LOG_PATTERN = "%d{ISO8601} - %-5p [%t:%c{40}@%L] - %m%n";
```

Comment:
this is used by the LogHandler endpoint, so this will only be reflected in the UI and REST calls and not the logs we write to the files.